docs/introduction.md

```diff
@@ -21,13 +21,9 @@ __SuperBench__ is a validation and profiling tool for AI infrastructure, which s
 It includes micro-benchmark for primitive computation and communication benchmarking,
 and model-benchmark to measure domain-aware end-to-end deep learning workloads.
 
-:::note
-SuperBench is in the early pre-alpha stage for open source, and not ready for general public yet.
-If you want to jump in early, you can try building latest code yourself.
-:::
 
 ## Overview
 
 The following figure shows the capabilities provide by SuperBench core framework and its extension.
 
-![SuperBench Structure](./assets/superbench_structure.png)
```

Comment:
remove the old image file?

Reply:
Sure. Changed to the new version. I will remove the .png file.